frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboard.tsx

```diff
@@ -31,16 +38,75 @@ type StateProps = {
 };
 
 type Props = MonitoringDashboardProps & StateProps;
+type watchResourcesType = { [key: string]: K8sResourceKind };
+type filterOption = { key: string; value: { type: string; option: string } & SelectOptionObject };
+enum OptionTypes {
+  selectAll = '#SELECT_ALL_WORKLOADS#',
+}
+const OptionValues = {
+  [OptionTypes.selectAll]: 'Select all',
```

Comment:
As per the UX design doc this should be ```All Workloads```.

Reply:
> Can we add query params to URL when the user in the context of workload?

@vikram-raj Updated the code as per design changes and added the query params in the URL to preserve the context, so now the user can  refresh the page and still be in the last context as well.